forest/daemon/src/daemon.rs

```diff
@@ -437,6 +440,14 @@ fn chain_path(config: &Config) -> PathBuf {
     PathBuf::from(&config.client.data_dir).join(&config.chain.name)
 }
 
+fn get_actual_network_name(internal_network_name: &str) -> String {
```

Comment:
Do we need to create an **owned** `String` every time this is called or can we instead return a `&str` with proper lifetime (alternatively a `Cow`).

Reply:
fixed